shared/src/libs/enums/icon-name.enum.ts

```diff
@@ -0,0 +1,6 @@
+const IconName = {
```

Comment:
It should be iconNameToIcon - the value is React Component of .svg file

Reply:
@v0ldemar01  can i save icons and this enum in frontend folder?

Reply:
Sure, it's only for frontend

Reply:
done